### PR TITLE
Adds Apitoxin, an admeme only chemical that turns people into bees

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

Adds apitoxin, an admeme only chemical that gives people the disease "Beesease The Sequel", which turns them into a bee
:bee: 
## Why It's Good For The Game

it makes admins buzz with happiness :bee: :bee: :bee: 

## Changelog
:cl: Fikou
add: A scientist from Nanotrasen just found a new chemical! Unfortunately, after ingestion, he turned into a bee
/:cl:

### CODE REVIEW

**Comment by Rowelll** [comment]
🐝

**Comment by Fikou** [comment]
:bee:

**Comment by MrPerson** [comment]
Immediately correct the spelling in your changelog, please.

Also what's the over/under on how long this stays admin-only, 3 months?

**Comment by Tlaltecuhtli** [comment]
the github ruleset says that any pr that changes the availability of species needs a prior authorization, and as far as i can see bee is a spices

**Comment by Fikou** [comment]
> the github ruleset says that any pr that changes the availability of species needs a prior authorization, and as far as i can see bee is a spices

bees are death machines not species

**Comment by Fikou** [comment]
> Also what's the over/under on how long this stays admin-only, 3 months?

probably depends if its abuzz with activity

**Comment by XDTM** [comment]
> 
> 
> > the github ruleset says that any pr that changes the availability of species needs a prior authorization, and as far as i can see bee is a spices
> 
> bees are death machines not species

not species, but they _are_ very spicy

**Comment by Suicidalpickles** [comment]
hip

**Comment by wesoda25** [comment]
hahhahah bee i get it very funny PLEASE speedmerge

**Comment by PKPenguin321** [comment]
lol....... 🅱️.......

**Comment by zxaber** [comment]
The question is, do they become bees that inject Apitoxin?

**Comment by ExcessiveUseOfCobblestone** [comment]
Admins can already turn people into bees?

**Comment by wesoda25** [comment]
As for real feedback its pointless code bloat for something that any admin could do with any knowledge of their buttons, all for a stupid fucking dead meme.

**Comment by optimumtact** [comment]
Why both a reagent and a disease

**Comment by Fikou** [comment]
> Why both a reagent and a disease

its a transformation disease, the borg one, gondola one, xeno one, all use them, snail disease and fungal tb use a reagent for the disease too

**Comment by nemvar** [comment]
I don't quite get the point of this if it's admin only. Admins can always just spawn a varedited transformation disease bottle.

**Comment by ATH1909** [comment]
Could you make the chem of the resulting bee be of a random type (like with bees from the holodeck?)?

**Comment by ATH1909** [comment]
> I don't quite get the point of this if it's admin only. Admins can always just spawn a varedited transformation disease bottle.

This one has some nice transformation messages.

Let the man have his bee-related fun.

**Comment by wesoda25** [comment]
Its hip to hate bees

**Comment by Fikou** [comment]
> Could you make the chem of the resulting bee be of a random type (like with bees from the holodeck?)?

holodeck bees are random from a list of toxin reagents 
but i can make the bee bee a random type if it wouldnt be too bad

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1730,10 +1730,20 @@
 	if(method==PATCH || method==INGEST || method==INJECT || (method == VAPOR && prob(min(reac_volume,100)*(1 - touch_protection))))
 		L.ForceContractDisease(new /datum/disease/transformation/gondola(), FALSE, TRUE)
 
+/datum/reagent/apitoxin
+	name = "Apitoxin"
+	description = "A strange liquid in a form similar to honey."
+	color = "#d3a308"
+	taste_description = "bad decisions, and honey"
```
Comment (81Denton) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: No need for the "and" here

**Comment by optimumtact** [comment]
get an admin to request this and you might have a case

**Pull request closed without merging**